evaluator/evaluator.py

```diff
@@ -354,6 +354,7 @@ def _vmaas_request_cves(updates_json):
     def evaluate_vmaas(self, system_platform):
         """Evaluates messages received from vmaas"""
         inventory_id = system_platform[0]
+        LOGGER.info("Evaluating vulnerabilities for inventory_id: %s", inventory_id)
```

Comment:
log-info-ing twice for every system feels like it's going to totally deluge the logfiles. Maybe debug() these instead?

Reply:
```
2019-02-12` 18:04:24,066:INFO:__main__:Received message type: upload_new_file
2019-02-12 18:04:24,067:INFO:__main__:Evaluating vulnerabilities for inventory_id: 9c79efcc-8f9a-47c7-b0f2-142ff52e89e9
2019-02-12 18:04:24,150:INFO:__main__:Finished evaluating vulnerabilities for inventory_id: 9c79efcc-8f9a-47c7-b0f2-142ff52e89e9
```

Now it looks like this for single evaluation, I think it's not that bad, but how to improve it and keep also the inventory_id of processed system in logs? E.g. make the second log as debug?

Reply:
Hm - logging 'i evaluated X' is def important - I'm just thinking about what happens when we have, say 100K inventory-ids, and we get a cve-update.

In any event - yeah, let's make the 'finished' a debug-only

Reply:
right but I assume there will be a lot of places to fix to reduce log size when handling 100K systems

I changed it here